llama-index-integrations/storage/chat_store/llama-index-storage-chat-store-dynamodb/llama_index/storage/chat_store/dynamodb/base.py

```diff
@@ -182,6 +208,12 @@ def set_messages(self, key: str, messages: List[ChatMessage]) -> None:
             Item={self.primary_key: key, "History": _messages_to_dict(messages)}
         )
 
+    async def aset_messages(self, key: str, messages: List[ChatMessage]) -> None:
+        self.is_async_initialized()
```

Comment:
should this just call `self.init_async_table()` instead?

Reply:
Oh right, that would be better